charts/rstudio-workbench/templates/configmap-general.yaml

```diff
@@ -12,11 +12,12 @@
   {{- $defaultOverrides = concat $defaultOverrides ( list $sessionVolumeOverride $sessionVolumeMountOverride ) }}
 {{- end }}
 {{- $defaultProfiles := dict "default-container-image" (first $defaultImages) "container-images" $defaultImages "allow-unknown-images" 1 }}
-{{- if $defaultOverrides }}
-  {{/* if defaultOverrides exist - add them */}}
-  {{- $defaultProfiles = mergeOverwrite $defaultProfiles ( dict "job-json-overrides" $defaultOverrides ) }}
-{{- end }}
 {{- $defaultProfilesConfig := dict "*" $defaultProfiles }}
+{{- /* Define the default rserver values - point to the launcher and IDE services in Kubernetes */}}
+{{- $defaultIDEServiceName := include "rstudio-workbench.fullname" . }}
+{{- $defaultIDEServiceURL := printf "http://%s.%s.svc.cluster.local:80" $defaultIDEServiceName $.Release.Namespace }}
+{{- $defaultRServerConfigValues := dict "launcher-sessions-callback-address" ($defaultIDEServiceURL) }}
```

Comment:
This solves an issue that @nahara7 was looking into to fix the callback address in general so that the verify-installation test works correctly!